test/helpers/cons.go

```diff
@@ -174,7 +174,7 @@ const (
 	KubectlPolicyNameLabel      = k8sConst.PolicyLabelName
 	KubectlPolicyNameSpaceLabel = k8sConst.PolicyLabelNamespace
 
-	CiliumStableVersion      = "v1.4"
+	CiliumStableVersion      = "v1.5"
```

Comment:
Maybe it's better if you cherry-pick the commit https://github.com/cilium/cilium/pull/8087/commits/196c182c938ff8c80e3bb577618bfd7ff469aac9

Reply:
Good idea, going to cherry-pick after I've fixed that PR.